src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -188,6 +189,24 @@ public boolean overwriteTombstones() {
     }
 
     /**
```

Comment:
You should add `includeVersions` to [getMap()](https://github.com/bbpennel/fcrepo-import-export/blob/3e82757cfd76adedbfa9d0a919a30859bf01104c/src/main/java/org/fcrepo/importexport/common/Config.java#L465)

..and update the following two tests as well:
- [testSerializedConfigCustom](https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/test/java/org/fcrepo/importexport/ArgParserTest.java#L319)
- [testSerializedConfigDefault](https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/test/java/org/fcrepo/importexport/ArgParserTest.java#L346)